lib/l10n/app_pl.arb

```diff
@@ -0,0 +1,260 @@
+  "error": "Błąd {error}",
+  "title": "Tytuł",
+  "time": "Czas",
+  "more_actions": "Więcej akcji",
+  "download_count": "Pobrane ({count})",
+  "add_count_to_playlist": "Dodaj ({count}) do Playlisty",
+  "add_count_to_queue": "Dodaj ({count}) do Kolejki",
+  "play_count_next": "Odtwórz ({count}) następne",
+  "album": "Album",
+  "copied_to_clipboard": "Skopiowano {data} do schowka",
+  "add_to_following_playlists": "Dodano {track} do danych Playlist",
+  "add": "Dodaj",
+  "added_track_to_queue": "Dodano {track} do kolejki",
+  "add_to_queue": "Dodano do kolejki",
+  "track_will_play_next": "{track} następny",
+  "play_next": "Odtwórz następny",
+  "removed_track_from_queue": "Usunięto {track} z kolejki",
+  "remove_from_queue": "Usunięto z kolejki",
+  "remove_from_favorites": "Usunięto z ulubionych",
+  "save_as_favorite": "Zapisz do ulubionych",
+  "add_to_playlist": "Dodaj do playlisty",
+  "remove_from_playlist": "Usuń z playlisty",
+  "add_to_blacklist": "Dodaj do czarnej listy",
+  "remove_from_blacklist": "Usuń z czarnej listy",
+  "share": "Udostępnij",
+  "mini_player": "Mały odwarzacz",
+  "slide_to_seek": "Przesuń, aby przewinąć do przodu lub do tyłu.",
+  "shuffle_playlist": "Odtwarzaj losowo z  playlisty",
+  "unshuffle_playlist": "Nie odtwarzaj losowo z  playlisty",
+  "previous_track": "Poprzedni utwór",
+  "next_track": "Następny utwór",
+  "pause_playback": "Zatrzymaj odwarzanie",
+  "resume_playback": "Wznów odwarzanie",
+  "loop_track": "Zapętl utwór",
+  "repeat_playlist": "Powtarzaj playlistę",
+  "queue": "Kolejka",
+  "alternative_track_sources": "Alternatywne źródła utworów",
+  "download_track": "Pobierz utwór",
+  "tracks_in_queue": "{tracks} utworów w kolejce",
+  "clear_all": "Wyczyść wszystko",
+  "show_hide_ui_on_hover": "Pokaż/Ukryj unoszący się interfejs",
+  "always_on_top": "Zawsze na wierzchu",
+  "exit_mini_player": "Opuść Mały odtwarzacz",
+  "download_location": "Zmień lokalizację",
+  "account": "Konto",
+  "login_with_spotify": "Zaloguj się używając konta Spotify",
+  "connect_with_spotify": "Połącz z Spotify",
+  "logout": "Wyloguj",
+  "logout_of_this_account": "Wyloguj z tego konta",
+  "language_region": "Język i Region",
+  "language": "Język",
+  "system_default": "Domyślny systemowy",
+  "market_place_region": "Region Rynku",
+  "recommendation_country": "Kraj rekomendacji",
+  "appearance": "Wygląd",
+  "layout_mode": "Tryb Układu",
+  "override_layout_settings": "Nadpisz responsywne ustawienia trybu układu",
+  "adaptive": "Adaptacyjny",
+  "compact": "Kompaktowy",
+  "extended": "Rozszerzony",
+  "theme": "Motyw",
+  "dark": "Ciemny",
+  "light": "Jasny",
+  "system": "Systemowy",
+  "accent_color": "Kolor Akcentu",
+  "sync_album_color": "Synchronizuj kolor albumu",
+  "sync_album_color_description": "Używa dominującego koloru okładki albumu jako koloru akcentującego",
+  "playback": "Odtwarzanie",
+  "audio_quality": "Jakość dźwięku",
+  "high": "Duża",
+  "low": "Mała",
```

Comment:
If this is related to audio quality, then it should be:

```suggestion
  "high": "Wysoka",
  "low": "Niska",
```

Reply:
That's better looks in create playlist ;)

Reply:
But that's also related to audio quality 🤷‍♂️

Reply:
I guess we need another key then